src/Apps/Settings/Routes/MyCollection/Components/MyCollectionAppDownload.tsx

```diff
@@ -9,16 +9,59 @@ import {
   Text,
 } from "@artsy/palette"
 import { resized } from "Utils/resized"
+import { Media } from "Utils/Responsive"
 
-export const MyCollectionAppDownload: React.FC = () => {
-  const image = resized(
-    "https://files.artsy.net/images/my-collection-get-app-img.jpg",
-    {
-      width: 910,
-      height: 652,
-    }
-  )
+const image = resized(
+  "https://files.artsy.net/images/my-collection-get-app-img.jpg",
+  {
+    width: 910,
+    height: 652,
+  }
+)
+
+const AppStoreBadges = () => (
+  <Flex>
+    <Box
+      width={120}
+      height={40}
+      as="a"
+      // @ts-ignore
+      href="https://apps.apple.com/us/app/artsy-buy-sell-original-art/id703796080"
```

Comment:
👀

Reply:
Yeah, but I don't think it's a good practice because we have a component for links like this. Especially if we have to add a `// ts-ignore` to it.

Reply:
Thanks. But which existing component is suitable for this? I am not aware of any besides good old `<div>`

Reply:
What about `RouterLink`?

Reply:
I didn't know we could use RouterLink for external urls.

Reply:
@damassi `RouterLink` for external urls, right?

Reply:
Yup! it will fall back to a regular link if it can't find the url passed to it in our router link list. See https://github.com/artsy/force/blob/main/src/System/Router/RouterLink.tsx#L31-L45